go/consensus/tendermint/tendermint.go

```diff
@@ -446,6 +446,24 @@ func (t *tendermintService) StateToGenesis(ctx context.Context, blockHeight int6
 	}, nil
 }
 
+func (t *tendermintService) GetGenesisDocument(ctx context.Context) (*genesisAPI.Document, error) {
+	genesisFileProvider, err := file.DefaultFileProvider()
```

Comment:
Why not just return `t.genesis`?

Also I now see that `StateToGenesis` also incorrectly uses `file.DefaultFileProvider`. While we're here it should be fixed to use `t.genesis` instead.

Reply:
I lazily copied it from `StateToGenesis`, fixed both now :)